src/libopenrave/plugindatabase.h

```diff
@@ -441,6 +441,13 @@ class RaveDatabase : public boost::enable_shared_from_this<RaveDatabase>
         if( pOPENRAVE_PLUGINS != NULL ) {
             utils::TokenizeString(pOPENRAVE_PLUGINS, delim, vplugindirs);
         }
+        for(int i=vplugindirs.size()-1;i>0;i--){
+            int j=0;
+            for(;j<i;j++){
+                if(vplugindirs[i]==vplugindirs[j])break;
```

Comment:
@cielavenir please use `{}` and new lines and instead of single char variables like `i`, how about `iplugindir`. thanks~